content/rs/networking/port-configurations.md

```diff
@@ -57,6 +57,39 @@ rladmin cluster config cm_port <new-port>
 After changing the Redis Enterprise Software web UI port, you must connect any new node added to the cluster to the UI with the custom port number:
 `https://newnode.mycluster.example.com:`**`<nonstandard-port-number>`**
 
+## Change the envoy  ports
+
+For system health monitoring, Redis uses the following ports by default:
+
+- Port 8002 for envoy admin
+
+- Port 8004 for envoy management server
+
+- Port 8006 for gossip envoy admin
+
+You can change each envoy port to a custom port using the [`rladmin cluster config`]({{<relref "/rs/references/cli-utilities/rladmin/cluster/config">}}) command as long as the new port is not in use by another process. When you change `envoy_admin_port`, expect a restart of envoy.
+
+To change the envoy admin port, run:
```

Comment:
I'm not sure you need "run" in these, but that's really more of a personal style issue. You know your audience best.